Source/AwsCommonRuntimeKit/crt/ByteBuffer.swift

```diff
@@ -308,22 +311,22 @@ extension ByteBuffer: AwsStream {
     }
 }
 
-extension ByteBuffer {
+public extension ByteBuffer {
```

Comment:
this is risky as it will put all methods public by default and can lead to accidental public visibility of an internal only method/prop.

default is internal which is inside module only.